eq-author/src/App/questionPage/Design/QuestionPageEditor/withHandleTransitionEnd.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const withHandleTransitionEnd = WrappedComponent => {
+  return class extends React.Component {
+    handleTransitionEnd = selector => {
+      document.querySelector(`[id="${selector}"`).focus();
```

Comment:
This feels like a quick fix instead of having all the field inputs work the same way. IMO it should all work like the RTE.

Reply:
I spent some time trying to get a consistent method to work for both RTE and normal fields. Feel free to have a go too! The API's are too inconsistent but I may have missed something 👍

Reply:
Maybe findDOMNode is better - but is still discouraged. 
https://reactjs.org/docs/react-dom.html#finddomnode

I think our only option is to add a focus method to all of the inputs types so we can call it with the ref like RTE.

Reply:
Yeah not a fan of this either and `finddomnode` not ideal either. 

Will have a look at some options...

Reply:
As discussed we will leave as-is and tackle this as tech debt (alongside other areas which handle `focus` in a similar way)